app/templates/content/index.hbs

```diff
@@ -90,19 +90,25 @@
                         </button>
                     </div>
 
-                    {{#if model.links.preprint_doi}}
-                        <div class="p-t-xs">
-                            <h4 class="p-v-md f-w-md"><strong>{{t "content.preprint_doi"}}</strong></h4>
+                    <div class="p-t-xs">
+                        <h4 class="p-v-md f-w-md"><strong>{{t "content.preprint_doi"}}</strong></h4>
+                        {{#if model.preprintDoiUrl}}
                             {{#if model.preprintDoiCreated}}
                                 <a href={{model.links.preprint_doi}} onclick={{action "click" "link" "Content - Preprint DOI" model.links.preprint_doi}}>{{extract-doi model.links.preprint_doi}}</a>
```

Comment:
When I mentioned updating the whole section to use `preprintDoiUrl`, I meant the code you didn't write, too, so it's consistent. If you don't mind?